src/components/formElements/MultiselectCustom.vue

```diff
@@ -0,0 +1,155 @@
+<script>
+import Multiselect from '@vueform/multiselect';
+
+export default {
+  components: {
+    Multiselect,
+  },
+  props: {
+    value: [Array, String],
+    options: Array,
+    placeholder: String,
+    noOptionsText: String,
+    disabled: Boolean,
+    mode: {
+      type: String,
+      default: 'single',
+    },
+    searchable: {
+      type: Boolean,
+      default: true,
+    },
+    createTag: {
+      type: Boolean,
+      default: true,
+    },
+    canClear: {
+      type: Boolean,
+      default: true,
+    },
+  },
+  emits: [
+    'select',
+    'deselect',
+    'clear',
+  ],
+  methods: {
+    select(tag) {
+      this.$emit('select', tag);
+    },
+    deselect(tag) {
+      this.$emit('deselect', tag);
+    },
+    clear() {
+      this.$emit('clear');
+    },
+  },
+};
+</script>
+
+<template>
+  <Multiselect
+    :class="`${globalTheme}-theme`"
+    :value="value"
+    :mode="mode"
+    :placeholder="placeholder"
+    :no-options-text="noOptionsText"
+    :options="options"
+    :searchable="searchable"
+    :create-tag="createTag"
+    :can-clear="canClear"
+    :disabled="disabled"
+    @select="tag => select(tag)"
+    @deselect="tag => deselect(tag)"
+    @clear="clear"
+  />
+</template>
+
+<style lang="scss">
+@import "../../../node_modules/@vueform/multiselect/themes/default.css";
```

Comment:
why is it needed? isn't it do it without import?

Reply:
you mean styles from this file are applied to Multiselect without import?
unfortunately, they are not :(

Reply:
ok )